isis/CMakeLists.txt

```diff
@@ -520,6 +520,8 @@ add_custom_target(appdata ALL COMMAND ${CMAKE_COMMAND} -E copy_if_different
   ${CMAKE_SOURCE_DIR}/appdata/translations/*.typ ${CMAKE_SOURCE_DIR}/src/*/apps/*/*.typ
   ${CMAKE_SOURCE_DIR}/src/*/apps/*/*.trn  ${CMAKE_SOURCE_DIR}/src/*/translations/*.trn
   ${CMAKE_SOURCE_DIR}/src/*/apps/*/*.pft ${CMAKE_SOURCE_DIR}/src/*/apps/*/*.def
+  ${CMAKE_SOURCE_DIR}/appdata/translations/*.def ${CMAKE_SOURCE_DIR}/appdata/translations/*.pvl
+#  ${CMAKE_SOURCE_DIR}/src/*/apps/*/*.pvl
```

Comment:
Is the comment because there are not any yet?

Reply:
Yes,  I think you mentioned that this needed to be added, but I don't remember the context.